.github/workflows/lint.yml

```diff
@@ -2,7 +2,7 @@ name: Lint
 on: [pull_request]
 
 jobs:
-  inspect-storage:
+  lint:
```

Comment:
should this be another PR?

Reply:
yes - will do

Reply:
done in https://github.com/ourzora/zora-protocol/pull/391